docker-compose.yaml

```diff
@@ -0,0 +1,56 @@
+version: "3.3"
+
+volumes:
+  logs: {}
+  dbdata: {}
+
+networks:
+  backend:
+
+services:
+  mining-pool:
+    image: mauroferra/mining-pool:v1.3.0
```

Comment:
We need to change the prefix `mauroferra`?

Reply:
It points to an image that I built with the Dockerfile included, which fetches the code from a release tarball, because the official mining-pool image is quite outdated

Reply:
We will not include an unofficial docker image in the reference code. Could you remove the image name and ask people to build the docker manually?

Reply:
ok, i can do that, there's the Dockerfile included just for that. I'll make 2 versions, one that uses the current context (useful for building an image from master with latest code) and one that pulls the zip from the latest release, just like it is now.

Will do in a bit today.

Reply:
Take your time, thanks!

Reply:
Done!